Description: relax dependency on indexmap
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-02-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -14,7 +14,7 @@
 fern = { version = ">= 0.5.0, < 0.7.0", features = ["colored"] }
 globset = "0.4.8"
 ignore = "0.4.18"
-indexmap = { version = "1.8.0", features = ["serde"] }
+indexmap = { version = "1.7", features = ["serde"] }
 itertools = ">= 0.9.0, < 0.11.0"
 log = "0.4.14"
 md5 = "0.7.0"
